### PR TITLE
build(dashmate): prepare MacOs build for notarization

### PR DESCRIPTION
**Pull request by shuplenkov**
## Issue being fixed or feature implemented

`cpu-features` dashmate sub dependency prevents MacOs installer notarization.

## What was done?

Removed unnecessary `cpu-features` sub dependency

## How Has This Been Tested?

Manual

## Breaking Changes

No

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/dashmate/package.tgz
Comment (shumkov) [inline packages/dashmate/package.tgz:1]: Do we need this?

**Review by shumkov — CHANGES_REQUESTED** [review]

File: packages/dashmate/package.tgz
Comment (shuplenkov) [reply packages/dashmate/package.tgz:1]: my bad

File: packages/dashmate/package.json [hunk 1]
```diff
@@ -103,6 +103,9 @@
     "sinon": "^11.1.2",
     "sinon-chai": "^3.7.0"
   },
+  "resolutions": {
```
Comment (shumkov) [inline packages/dashmate/package.json:106]: I think `resolutions` works only in root package.json

Comment (shuplenkov) [reply packages/dashmate/package.json:106]: But when we build dashmate, this files becomes our root package

**Review by shumkov — APPROVED** [review]
👍

File: packages/dashmate/package.json (hunk 1 above)
Comment (shumkov) [reply packages/dashmate/package.json:106]: Shall we remove it then from the root one?

**Pull request merged**